docs/conf.py

```diff
@@ -52,7 +52,7 @@
 
 # General information about the project.
 project = 'SoS'
-copyright = '2014, Bryn Reeves'
+project_copyright = '2014, Bryn Reeves'
```

Comment:
`project_copyright` is just an alias for `copyright` but using the former avoids the redefinition conflict with builtin `copyright`.